src/TConsole.cpp

```diff
@@ -1032,7 +1032,6 @@ void TConsole::changeColors()
     else if( mIsSubConsole )
     {
         mDisplayFont.setStyleStrategy( (QFont::StyleStrategy)(QFont::NoAntialias | QFont::PreferQuality ) );
-#if defined(Q_OS_MAC) || (defined(Q_OS_LINUX) && QT_VERSION >= 0x040800)
```

Comment:
This should be `#if defined(Q_OS_MAC) || defined(Q_OS_LINUX)` just like below, right?